src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -137,7 +126,7 @@ public static File renderTileMetricsFileFromBasecallingDirectory(final File illu
     ) throws FileNotFoundException {
         final Map<Integer, Map<Integer, Collection<TilePhasingValue>>> phasingValues = getTilePhasingValues(phasingMetricsFiles, readStructure);
         for (File tileMetricsOutFile : tileMetricsOutFiles) {
-            final TileMetricsOutReader tileMetricsIterator = new TileMetricsOutReader(tileMetricsOutFile, TileMetricsOutReader.TileMetricsVersion.THREE);
+            final TileMetricsOutReader tileMetricsIterator = new TileMetricsOutReader(tileMetricsOutFile);
```

Comment:
Do you still need the V3 in the method name here?  That is - will this now work with V2 and V3?